### ISSUE TITLE
Configure Traffic Cop for concurrent experiments

### ISSUE THREAD

**Issue by ejregithub** [mozilla/bedrock#6920]
## Description

Run 
 exp01 - Headline Test on /new: Replace the headline “The new Firefox” #6851 
and  
Experiment: Dry market test for new Desktop pitches #6899 
concurrently

https://github.com/mozilla/bedrock/compare/master...jpetto:multiple-experiments-same-page?expand=1

### Estimates

| Estimates |  |
| ------------- | ------------- |
| TL | |
| PM | |
| ENG | |
| SEO | |
| ANA | |
| DES | |
| COPY | |

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] This is an example of a success criteria. Use a checklist!
- [ ] This is an example of a success criteria.

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [ ] This is an example of a risk. Use a checklist!
- [ ] This is an example of a risk.

#### Tasks
- [ ] This is an example of a task. Use a checklist!
- [ ] This is an example of a task.

**Comment by jpetto** [comment]
Admitting I do not know the specifics of the experiments, but here is a basic branch running three different experiments on the same page:

https://github.com/mozilla/bedrock/compare/master...jpetto:multiple-experiments-same-page?expand=1

Hopefully it's useful!

**Comment by ejregithub** [comment]
@alexgibson - please review JPetto's work here for possible use in running experiments concurrently on /new.  Per @hoosteeno comments in Slack:

"We can do this, but we must not block dry market test to make it happen."

**Comment by alexgibson** [comment]
Thanks for the pseudo code @jpetto 👍 

This may not quite work as-is, since one experiment is using redirects and the code exists in the `<head>` of the page, and the other is replacing content in the DOM and exists in the `<body>` of the page. That said, I'm usre we can work out something similar that checks for the existence of a cookie.

Either way, I'm blocked on doing this work until both https://github.com/mozilla/bedrock/pull/6908 and https://github.com/mozilla/bedrock/pull/6925 can be merged to master.

I think Craig is also now out on PTO, so I'm wondering if I should close his PR and re-issue a new one?

**Comment by alexgibson** [comment]
I've taken Craig's work and moved it to a new PR so that he can enjoy his PTO: https://github.com/mozilla/bedrock/pull/6926

I'll work on getting both PRs to play nicely in tandem.

**Comment by alexgibson** [comment]
I think I have some code in place to handle this, using the following logic:

- Visitor hits `/en-US/firefox/new/`
  - Dry market experiment config runs:
    - If user has existing cookie from competing headline experiment, do nothing.
    - If user meets experiment criteria, enter into the experiment (at a 50% sample rate).
    - If not within the 50% sample rate, do nothing.
  - Headline experiment config runs:
    - If user has existing cookie from competing dry-market experiment, do nothing. 
    - Else if user meets experiment criteria, enter into the experiment

**Comment by alexgibson** [comment]
Things I need to happen in order for the above to work:

1. Someone needs to review and merge headline experiment PR: https://github.com/mozilla/bedrock/pull/6908
2. I then need to rebase the dry-market experiment, before I can mark it as ready for code review.
3. Once that is done, someone needs to review the dry-market PR (and also to confirm that the experiments don't conflict).

**Comment by alexgibson** [comment]
This is now done, will enable both experiments today.
